src/lib/parseSkyFromScene.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file parseSkyFromScene.js
+ * Exports a helper menthod that assists in fetching the sky for a scene.
+ */
+
+/**
+ * Helper menthod that assists in fetching the sky for a scene.
+ *
+ * @param {object} scene
+ *   Experience object, containing scenes.
+ * @param {bool} thumbnail
+ *   Boolean indicating whether or not the returned value should be a thumbnail.
+ *
+ * @returns {object}
+ *   Object containing a url, and type properties that indicate the type of sky
+ *   and the path to it's file.
+ */
+const parseSkyFromScene = (scene = null, thumbnail = false) => {
```

Comment:
This is a little helper lib  that can yank a sky out of a scene object. This is useful because skies can be pulled from two different fields, one for videospheres, one for photospheres. This also makes it easy to pull sky thumbnails, which can be a little messy.